internal/process_config.go

```diff
@@ -0,0 +1 @@
+package internal
```

Comment:
What is this file for?

Reply:
Oh, I guess I wanted to move something here and then didn't use it. I'll remove it!